meta-lmp-bsp/recipes-bsp/trusted-firmware-a/tf-a-fio-st.inc

```diff
@@ -22,3 +22,42 @@ TFA_PLATFORM  = "stm32mp1"
 TF_A_SUFFIX   = "stm32"
 TF_A_METADATA_JSON ?= "plat/st/stm32mp1/default_metadata.json"
 TF_A_SOC_NAME  = "${STM32MP_SOC_NAME}"
+
+do_deploy:append:stm32mp15-eval-sec() {
+    if [ -z "${STM32_CUBE_PATH}" ] || [ -z "${STM32_ROT_KEY_PATH}" ] || [ -z "${STM32_ROT_KEY_PASSWORD}" ]; then
```

Comment:
It might be better to isolate this logic in a function and call it when stm32mp15-eval-sec.

Just thinking on how a customer would reuse this with a custom board.